lib/anemone/core.rb

```diff
@@ -298,10 +298,10 @@ def skip_link?(link)
     #
     # Kills all active threads
     #
-  def shutdown
-      @tentacles.each {|t| t.kill rescue nil }
-      @pages = nil
-  end
+    def shutdown
+        @tentacles.each {|t| t.kill rescue nil }
```

Comment:
Sorry to nit pick, only lines 301 & 304 should have shifted.

[From CONTRIBUTING.md](https://github.com/rapid7/metasploit-framework/blob/master/CONTRIBUTING.md#code-contributions)  
* Do stick to the [Ruby style guide](https://github.com/rubocop-hq/ruby-style-guide#source-code-layout)

```
  def shutdown
    @tentacles.each {|t| t.kill rescue nil }
    @pages = nil
  end
```

Reply:
Sorry about that!